src/__tests__/__snapshots__/basic-test.js.snap

```diff
@@ -75,10 +81,10 @@ Foo.propTypes = {
     bar: require('prop-types').number.isRequired,
     baz: require('prop-types').oneOf(['literal']).isRequired
   }).isRequired,
-  some_external_type: typeof babelPluginFlowReactPropTypes_proptype_SomeExternalType === 'function' ? babelPluginFlowReactPropTypes_proptype_SomeExternalType.isRequired ? babelPluginFlowReactPropTypes_proptype_SomeExternalType.isRequired : babelPluginFlowReactPropTypes_proptype_SomeExternalType : require('prop-types').shape(babelPluginFlowReactPropTypes_proptype_SomeExternalType).isRequired,
+  some_external_type: typeof (_types.bpfrp_babelPluginFlowReactPropTypes_proptype_SomeExternalType || _propTypes2.default.func) === 'function' ? (_types.bpfrp_babelPluginFlowReactPropTypes_proptype_SomeExternalType || _propTypes2.default.func).isRequired ? (_types.bpfrp_babelPluginFlowReactPropTypes_proptype_SomeExternalType || _propTypes2.default.func).isRequired : _types.bpfrp_babelPluginFlowReactPropTypes_proptype_SomeExternalType || _propTypes2.default.func : require('prop-types').shape(_types.bpfrp_babelPluginFlowReactPropTypes_proptype_SomeExternalType || _propTypes2.default.func).isRequired,
   some_external_type_intersection: require('prop-types').shape(Object.assign({}, {
     foo: require('prop-types').string.isRequired
-  }, babelPluginFlowReactPropTypes_proptype_SomeExternalType === require('prop-types').any ? {} : babelPluginFlowReactPropTypes_proptype_SomeExternalType)).isRequired
+  }, (_types.bpfrp_babelPluginFlowReactPropTypes_proptype_SomeExternalType || _propTypes2.default.func) === require('prop-types').any ? {} : _types.bpfrp_babelPluginFlowReactPropTypes_proptype_SomeExternalType || _propTypes2.default.func)).isRequired
```

Comment:
some leftovers `require('prop-types')` are here, i guess its not intended

Reply:
Good catch, thanks.